app/models/bot/tagger.rb

```diff
@@ -0,0 +1,65 @@
+require 'json'
+
+class Bot::Tagger < BotUser
+  check_settings
+  class Error < ::StandardError
```

Comment:
**question** we create a custom error here, but don't seem to use it. is that intended?

Reply:
Was copied from AlegreBot without thinking :sweat: '.  Based on the comment below, we'll use this in the rescue blog of `run`. Thanks